### PR TITLE
feat(scheduler): metrics for all schedules

### PR DESCRIPTION
**Pull request by filletofish**
Class Schedule now provides static methods for accessing all schedules metrics with default configuration, such as `metricAllErrors` for viewing errors when executing targets.

Advances #23394

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-scheduler-alpha/README.md [hunk 1]
```diff
@@ -245,7 +245,16 @@ EventBridge Scheduler publishes additional metrics when your schedule exhausts i
...
 
-TODO: Not yet implemented. See section in [L2 Event Bridge Scheduler RFC](https://github.com/aws/aws-cdk-rfcs/blob/master/text/0474-event-bridge-scheduler-l2.md)
+Class `Schedule` provides static methods for accessing all schedules metrics with default configuration,
+ such as `metricAllErrors` for viewing errors when executing targets.
+
+ ```ts
+new cloudwatch.Alarm(this, 'SchedulesErrorAlarm', {
+    metric: Schedule.metricAllErrors(),
+    threshold: 0,
+    evaluationPeriods: 1
```
Comment (kaizencc) [inline packages/@aws-cdk/aws-scheduler-alpha/README.md:255]:
```suggestion
    evaluationPeriods: 1,
```

**Review by kaizencc — CHANGES_REQUESTED** [review]
Looks good @filletofish. Build is failing though, and can we also have 1 unit test just for completeness sake?

**Review by kaizencc — APPROVED** [review]
Yay @filletofish! Thanks!

**Comment by vinayak-kukreja** [comment]
@Mergifyio update

**Comment by vinayak-kukreja** [comment]
@Mergifyio update

**Review by vinayak-kukreja — APPROVED** [review]

**Pull request merged**